torchvision/ops/misc.py

```diff
@@ -142,15 +143,12 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
             missing_keys, unexpected_keys, error_msgs)
 
     def forward(self, x):
-        # move reshapes to the beginning
-        # to make it fuser-friendly
-        w = self.weight.reshape(1, -1, 1, 1)
-        b = self.bias.reshape(1, -1, 1, 1)
-        rv = self.running_var.reshape(1, -1, 1, 1)
-        rm = self.running_mean.reshape(1, -1, 1, 1)
-        scale = w * rv.rsqrt()
-        bias = b - rm * scale
-        return x * scale + bias
```

Comment:
Can you revert those changes? It was necessary to get much faster runtime when running with the JIT fuser (it was all done in a single kernel call), and I'm not sure the fuser is already smart enough to re-order those operations so that the we don't need to do it ourselves.

cc @suo for an example where we had to move operations to the beginning to facilitate fusion. Note that in this case one complicating factor is the reshapes and the broadcasting

Reply:
Yes I wasn't sure about JIT, thanks for the details. Just reverted the changes!